CODEOWNERS

```diff
@@ -282,7 +282,7 @@ Makefile* @cilium/build
 /Documentation/cmdref/
 /Documentation/community/community.rst @cilium/contributing
 /Documentation/community/governance/ @cilium/contributing
-/Documentation/community/governance/commit_access.rst @cilium/contributing @cilium/docs-structure
+/Documentation/community/governance/commit_access.rst @cilium/tophat
```

Comment:
TopHat isn't committers. Committers is `@cilium/committers`.

Reply:
Sorry didn't see that one